client/components/domains/transfer-domain-step/transfer-domain-options.jsx

```diff
@@ -0,0 +1,160 @@
+		domain: PropTypes.string,
+		onSubmit: PropTypes.func,
+	};
+
+	onClick = () => {
+		const { dnsImport, nameservers, privacy } = this.state;
+
+		this.props.onSubmit( this.props.domain, nameservers, privacy, dnsImport );
+	};
+
+	state = {
+		dnsImport: true,
+		nameservers: true,
+		privacy: true,
+	};
+
+	toggle( property, value ) {
+		this.setState( {
+			[ property ]: value,
+		} );
+	}
+
+	toggleNameservers = event => {
+		const value = event.target.checked;
+
+		this.toggle( 'nameservers', value );
+		if ( ! value ) {
+			this.toggle( 'dnsImport', false );
+		}
+	};
+
+	togglePrivacy = event => {
+		this.toggle( 'privacy', event.target.checked );
+	};
+
+	toggleDnsImport = event => {
+		this.toggle( 'dnsImport', event.target.checked );
+	};
+
+	render() {
+		const cost = this.props.products.private_whois
+			? this.props.products.private_whois.cost_display
+			: null;
+		const { translate } = this.props;
+
+		const headerLabel = translate(
+			"You have a few options for how to set up your domain once it's transferred. " +
+				'Review these options and choose the ones that apply to ensure that everything is ' +
+				'configured the way you want it.'
+		);
+
+		return (
+			<div className="transfer-domain-step__options">
+				<SectionHeader>{ headerLabel }</SectionHeader>
+				<Card>
+					<FormFieldset>
+						<FormLabel>
+							<FormCheckbox
+								onChange={ this.toggleNameservers }
+								checked={ this.state.nameservers }
+							/>
+							<span>
+								<FormLegend>{ translate( 'Manage your domain at WordPress.com.' ) }</FormLegend>
+								{ translate(
+									'{{strong}}Recommended!{{/strong}} ' +
+										"You'll be able to edit your domain settings, create email addresses with your " +
+										'domain and more, all from your WordPress.com dashboard.',
+									{
+										components: { strong: <strong /> },
+									}
+								) }
+							</span>
+						</FormLabel>
+					</FormFieldset>
+					<FormFieldset>
+						<FormLabel>
+							<FormCheckbox
+								disabled={ ! this.state.nameservers }
+								onChange={ this.toggleDnsImport }
+								checked={ this.state.dnsImport }
+							/>
+							<span>
+								<FormLegend>
+									{ translate( 'Use this domain with a WordPress.com site.' ) }
+								</FormLegend>
+								{ translate(
+									"Check this if you're ready to use this domain with a WordPress.com site. " +
+										'Once you do, any sites using this domain on another host will no longer be viewable ' +
+										'at this web address.',
+									{
+										components: { strong: <strong /> },
+									}
+								) }
+							</span>
+						</FormLabel>
+					</FormFieldset>
+					<FormFieldset>
+						<FormLabel>
+							<FormCheckbox onChange={ this.togglePrivacy } checked={ this.state.privacy } />
+							<span>
+								<FormLegend>
+									{ translate( 'Make contact information private.' ) }
+									{ cost &&
+										translate( '{{small}}+%(cost)s / year{{/small}}', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( '%(cost)s {{small}}/year{{/small}}' )` *ES Score: 19*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/style-transfer-precheck-screen)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).